doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -20,33 +19,27 @@ but they are probably less current than `-help` or this document).
 
 .. _interactive-use:
 
-Interactive use (coqtop)
-------------------------
+Interactive use (rocq repl)
+---------------------------
 
 In the interactive mode, also known as the Coq toplevel, users can
 develop their theories and proofs step by step. The Coq toplevel is run
```

Comment:
Remove the outdated introduction that this would be how users develop their theories.

```suggestion
The Rocq toplevel (or read-eval-print-loop) is run
```